addons/project/views/rating_views.xml

```diff
@@ -63,6 +90,13 @@
             <xpath expr="//pivot" position="attributes">
                 <attribute name="js_class">project_rating_pivot</attribute>
             </xpath>
+            <xpath expr="//pivot" position="inside">
+                <field name="res_id" invisible="1"/>
+                <field name="parent_res_id" invisible="1"/>
+            </xpath>
```

Comment:
The both fields are defined in the parent view. 🙂 
See [here](https://github.com/odoo/odoo/blob/85b8a951167511852b0d1ecd21a36b0aaee474d1/addons/rating/views/rating_rating_views.xml#L114-L115)

Reply:
Done.